internal/source/prometheus/source.go

```diff
@@ -0,0 +1,75 @@
+package prometheus
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/kubeshop/botkube/pkg/api"
+	"github.com/kubeshop/botkube/pkg/api/source"
+)
+
+const (
+	// PluginName is the name of the Prometheus Botkube plugin.
+	PluginName = "prometheus"
+)
+
+// Source prometheus source plugin data structure
+type Source struct {
+	prometheus    *Client
+	l             sync.Mutex
+	pluginVersion string
+	startedAt     time.Time
+}
+
+func NewSource(version string) *Source {
+	return &Source{
+		pluginVersion: version,
+		startedAt:     time.Now(),
+	}
+}
+
+// Stream streams prometheus alerts
+func (p *Source) Stream(ctx context.Context, input source.StreamInput) (source.StreamOutput, error) {
+	out := source.StreamOutput{Output: make(chan []byte)}
+	config, err := MergeConfigs(input.Configs)
+	if err != nil {
+		return source.StreamOutput{}, fmt.Errorf("while merging input configs: %w", err)
+	}
+	go p.consumeAlerts(ctx, config, out.Output)
+	return out, nil
+}
+
+// Metadata returns metadata of prometheus configuration
+func (p *Source) Metadata(_ context.Context) (api.MetadataOutput, error) {
+	return api.MetadataOutput{
+		Version:     p.pluginVersion,
+		Description: "Prometheus polls alerts from Prometheus alert manager to report in Botkube",
```

Comment:
```suggestion
		Description: "Prometheus plugin polls alerts from configured Prometheus Alertmanager.",
```

From what I see in the official doc, it should be written as `Alertmanager` instead of `alert manager`.